flink-ml-python/pyflink/ml/lib/feature/vectorassembler.py

```diff
@@ -27,21 +29,61 @@ class _VectorAssemblerParams(
     HasOutputCol,
     HasHandleInvalid
 ):
+
+    """
+    Checks the inputSizes parameter.
+    """
+    def sizes_validator(self) -> ParamValidator[Tuple[int]]:
+        class SizesValidator(ParamValidator[Tuple[int]]):
+            def validate(self, indices: Tuple[int]) -> bool:
+                if indices is None:
+                    return False
+                for val in indices:
+                    if val < 0:
+                        return False
+                return len(indices) != 0
+        return SizesValidator()
+
     """
     Params for :class:`VectorAssembler`.
     """
 
+    INPUT_SIZES: Param[Tuple[int, ...]] = IntArrayParam(
+        "input_sizes",
+        "Sizes of the input elements to be assembled.",
+        None,
+        sizes_validator(None))
```

Comment:
Would it be enough to use `SizesValidator()` directly, instead of adding a `sizes_validator` method?